### DISCUSSION TITLE
Font size in visual editor

### DISCUSSION THREAD

**Discussion by sashabel** [requarks/wiki#4538] [Error / Bug Report] [answered]
My host information:

```
OS: Ubuntu 18.04 and Docker
Wiki.js version: 2.5.219
Database engine: PostgreSQL 11
```

I created a page with a couple of lines, each line was created with a different 'Font Size' in the editing mode:
![Screenshot from 2021-10-03 11-46-03](https://user-images.githubusercontent.com/17266906/135746480-9365af7d-48d1-44a7-b74c-a03b4454b6b1.png)

In the presentation mode all the lines are represented with the same font size:

![Screenshot from 2021-10-03 11-45-23](https://user-images.githubusercontent.com/17266906/135746520-2d524d59-7916-45c9-bdd6-eea44e176ac8.png)

**Comment by MisterCookie1234** [comment]
Can Confirm this Bug on my Side and would love a fix for it.

**Comment by NGPixel** [comment, marked as answer]
Has been fixed in 2.5.264.
